src/gui/gui_outliner_window.hpp

```diff
@@ -430,8 +430,15 @@ class outliner_entry_text : public color_text_element {
 				color = text::text_color::white;
 				set_text(state, full_str);
 			} else if(fat_nf.get_promotion_type()) {
-				auto full_str = text::produce_simple_string(state, fat_nf.get_name()) + " (" + text::get_dynamic_state_name(state, siid) + ", "  + text::format_percentage(fat_si.get_demographics(demographics::to_key(state, fat_nf.get_promotion_type())) / fat_si.get_demographics(demographics::total)) + ")";
 				color = text::text_color::white;
+				//Is the NF not optimal? Recolor it
+				if(fat_nf.get_promotion_type() == state.culture_definitions.clergy) {
```

Comment:
This could've been remade to check if pop promotion into the promoting pop is even possible; although the performance impact of continuously computing pop promotion chance and checking if it's possible might be too much for a QoL feature.